src/main/java/seedu/address/model/hotel/bill/Bill.java

```diff
@@ -54,6 +55,13 @@ public void addRoomCost(RoomCost roomCost, Stay stay) {
     /**
      * Adds {@code roomCost} to the stored list of chargeable objects.
      */
+    public void addExtendRoomCost(RoomCost roomCost, Stay stay, LocalDateTime fromDate) {
+        charges.add(roomCost);
+        long daysOfStay = stay.getBookDuration().getDaysDiff(fromDate);
```

Comment:
does daysOfStay refer to the number of days the stay is extended by?

Reply:
Yes correct

Reply:
i think there is no need to add roomcost to charges again since they are staying in the same room?